seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -440,7 +441,7 @@
         "  <div class=\"cardContent\">",
         "    <div class=\"cardText\">",
         "      <strong><h4>Google</h4></strong>",
```

Comment:
Good point! If we change this to highlighting `"Stanford University"` we should remove the highlighting from the h4